src/chacha.rs

```diff
@@ -81,6 +81,22 @@ impl ChaChaRng {
 
     /// Create an ChaCha random number generator using the default
     /// fixed key of 8 zero words.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use rand::{Rng, ChaChaRng};
```

Comment:
Build fails without it - `next_u32` is from `rng`.

Only commenting again since the comment got lost after I pushed new code.